README.md

```diff
@@ -575,6 +575,7 @@ Comparison of NoSQL servers: http://kkovacs.eu/cassandra-vs-mongodb-vs-couchdb-v
 
 * [Tiger VNC](http://tigervnc.org/) - TigerVNC is a high-performance, platform-neutral implementation of VNC (Virtual Network Computing), a client/server application that allows users to launch and interact with graphical applications on remote machines.
 * [Remmina](http://www.remmina.org/wp/) - A Feature rich remote desktop application for linux  and other unixes.
+* [RustDesk](https://github.com/rustdesk/rustdesk) - Open source virtual / remote desktop infrastructure to display and control your PC and Android devices from anywhere at anytime.
```

Comment:
rustdesk is being added here https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3333

Can you please remove it from your patch?